i18n/changed.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Determine if the source translation files are up-to-date.
+"""
+from __future__ import print_function
+from subprocess import CalledProcessError
+import sys
+
+from i18n import Runner
+from i18n.execute import execute
+
+
+class Changed(Runner):
+    """
+    Class used to check if the source translation files are up-to-date
+    """
+    def run(self, args):
+        """
+        Main entry point of script
+        """
+        changes_detected = self.detect_changes()
+        message = self.get_message(changes_detected)
+        print(message)
+        sys.exit(1)
```

Comment:
#1 is not enough because it makes the code too brittle: there is no way to use this function in composable ways. If you look in main.py (https://github.com/edx/i18n-tools/blob/master/i18n/main.py#L63) you see the right way to do it: "main" functions should return a status code, and the very top level of the program should use sys.exit to use that value as the status code of the process. The last line of `main` (https://github.com/edx/i18n-tools/blob/master/i18n/main.py#L60) is helping with that by passing along the return value from `module.main()`, which in a twisty bit of programming, is invoking `Runner.__call__`.  That function calls your `run` method, but doesn't pass along the return value.  So we need to change that line (https://github.com/edx/i18n-tools/blob/master/i18n/__init__.py#L52) from `self.run(args)` to `return self.run(args)`.  Then instead of calling sys.exit directly, your `run` method can just return the status code.